exercises/pascals-triangle/src/example.c

```diff
@@ -0,0 +1,40 @@
+#include "pascals_triangle.h"
+
+void pt_free(size_t ** mem_block, size_t n)
+{
+   for (size_t i = 0; i < n; i++) {
+      free(mem_block[i]);
+   }
+   free(mem_block);
+}
+
+size_t **pascals_triangle(char *count)
+{
+   if (count == NULL) {
+      return (size_t **) NULL;
+   }
+   size_t **zero = (size_t **) calloc(1, sizeof(size_t *));
+   zero[0] = (size_t *) calloc(1, sizeof(size_t));
+   int i, j;
```

Comment:
rows is int , so i thought comparison between int and size_t will not be good , the reason i did not declare i , j in the loop is that , According to C89 standards its illegal , but exercism/xc is using C99 standards , so i will commit the changes soon